src/index.js

```diff
@@ -1,51 +1,62 @@
 /** @format */
 
-/** @format */
-
-// import _ from 'lodash';
 import './style.css';
+import setItem from './asset/js/storage.js';
+import {
+  addList, input,
+} from './asset/js/add.js';
+/* eslint-disable */
+import { showList } from './asset/js/showList.js';
 
-const todoListContainer = document.querySelector('#todo-list-container');
-class Todo {
-  constructor() {
-    this.userTodo = [
-      { idex: 0, description: 'Double-tab to edit', completed: false },
-      {
-        idex: 1,
-        description: "Drag'n drop to reorder your list",
-        completed: false,
-      },
-      {
-        index: 2,
-        description: 'Manage all your lists in one place',
-        completed: false,
-      },
-      {
-        idex: 3,
-        description: 'Resync to clear out the old',
-        completed: false,
-      },
-    ];
-  }
+let todoTask = JSON.parse(localStorage.getItem('todoTask')) || [];
+export default todoTask;
+/* eslint-disable */
```

Comment:
- [ ] as the previous CR already mentioned please do not disable Eslint it will help check for your Javascript code standards. Kindly enable Eslint and fix the errors manually.